circle.yml

```diff
@@ -13,7 +13,10 @@ dependencies:
     - mix compile
 test:
   override:
-    - mix test --include requires_env --cover
+    - |
+      if [ ${CIRCLE_PR_USERNAME} ]; then MIX_ENV=test mix test --cover; else MIX_ENV=test mix test --include requires_env --cover; fi
```

Comment:
This looks great. Can we add a line to docs explaining the change? Think there's à CI section.